engine/flink/management/src/test/scala/pl/touk/nussknacker/engine/management/FlinkRestManagerSpec.scala

```diff
@@ -275,8 +275,8 @@ class FlinkRestManagerSpec extends FunSuite with Matchers with PatientScalaFutur
     val version = 15L
     val user = "user1"
 
-    statuses = List(JobOverview(jid, processName.value, 40L, 10L, JobStatus.FINISHED.name()),
-      JobOverview("1111", "p1", 35L, 30L, JobStatus.FINISHED.name()))
+    statuses = List(JobOverview(jid, processName.value, 40L, 10L, JobStatus.FINISHED.name(), baseTasksOverview().copy(finished = 1)),
```

Comment:
maybe let's add more parameters (with defaults) to baseTasksOverview?

Reply:
changed